HTTP_200/settings.py

```diff
@@ -32,7 +32,7 @@
 # Application definition
 
 INSTALLED_APPS = (
-    # 'rest_framework',
+    'rest_framework',
```

Comment:
@shriyanka any reason for uncommenting this ?? I think we dont need it as of now.